docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,465 @@ import { StyledContent } from '../../components/styled.elements';
+
+### Overview
 
-The Redocly configuration file must be in YAML format.
+The `apis` section lets you configure one or more OpenAPI definition files. This gives you the flexibility to reference specific files in commands, and configure each file at a granular level.
 
-The contents of the file should conform to the following structure:
+Every file listed in this section is identified by `name@version`. `@version` is optional. It allows you to derive multiple API definition versions from the same root file. When a version is not provided, Redocly apps interpret a file as being `latest`. Every `name@version` combination must be unique.
+
+For every OpenAPI definition listed in the `apis` section, use the `root` property to define the path to the file. For example:
 
 ```yaml
-organization:
-# The organization ID of a Redocly Workflows organization.
-# Used for easier integration with the API registry and hosted Reference docs,
-# not required for local OpenAPI CLI usage.
 apis:
-# A collection of objects, each representing one API.
-# An API should be defined by a name, an optional version,
-# and the path to the root OpenAPI document.
-# Additionally, each API can have its own lint and features.openapi
-# sections for fine-grained control.
-# Example:
-  main:
+  main@v1:
     root: ./openapi/openapi.yaml
-    lint:
-      rules:
-        example-rule: warn
-  external@v1:
-    root: ./openapi/external.yaml
     labels:
-      - external
-    features.openapi:
-      hideLogo: true
+      - main
+    lint: []
+    features.openapi: []
+```
 
-lint:
-# Global configuration for the lint command.
+### Impact on commands
 
-features.mockServer:
-# Configuration for the optional mock server feature in the API registry.
-# Available in Pro and Enterprise plans only.
+You can pass `name` or `name@version` in commands like `lint` and `stats` to act only on the OpenAPI definition files you specify.
 
-region: eu
-# Configuration for optional AWS region selection when logging into Redocly services.
+Bonus: You only need to type `name` or `name@version` in commands, not the entire path to the file. No one's getting any younger — especially not us. Where we can save you time, we will!
 
-resolve:
-# Configuration for resolving external links in your definition that are not publicly accessible.
-# Not required for Redocly API registry links.
+### Summary of `apis` properties
 
-features.openapi:
-# The API reference docs options, including theme options.
+| Property |  | Type | Description |
+|---|---|---|---|
+| `{apiName}` |  | `patternProperties` | REQUIRED. Identifies an API definition file. |
+| `{apiName}@{apiVersion}` |  | `patternProperties` | Identifies multiple versions of an API definition file. `{apiName}` is required but `@version` is optional. |
+|  | `root` | `string` | REQUIRED. The relative path to where a definition file is stored. |
+|  | `labels` | `array` | Assigns Redocly Workflows organization-wide labels to a definition file. |
```

Comment:
suggestion: can we link from this to the section that describes `labels` in more detail? And then do the same for every property/section where it applies.

Reply:
Sure. Do I express this as:

[link text](#labels)
See the [`labels`](#labels) section for more information.

Reply:
Yes - great idea. Is this the correct syntax:

`[labels](./configuration-file.mdx#labels)`

Reply:
For links within the same file, the file name is not needed. You can directly link to the heading, like this: [labels](#labels)